v2/futures/account_service.go

```diff
@@ -106,4 +106,8 @@ type AccountPosition struct {
 	PositionInitialMargin  string `json:"positionInitialMargin"`
 	Symbol                 string `json:"symbol"`
 	UnrealizedProfit       string `json:"unrealizedProfit"`
+	EntryPrice             string `json:"entryPrice"`
+	MaxNotional            string `json:"maxNotional"`
+	PositionSide           string `json:"positionSide"`
```

Comment:
PositionSide should be of type PositionSideType 

https://github.com/adshao/go-binance/blob/8754a53fcf6a850b3f3f93c220174c9648ec8166/v2/futures/client.go#L25

Reply:
Great, I have made the change. Thanks for the review!